lib/index.js

```diff
@@ -0,0 +1,229 @@
+// Dependencies
+var GitLogParser = require("gitlog-parser").parse
+  , Ul = require("ul")
+  , Typpy = require("typpy")
+  , Events = require("events")
+  , EventEmitter = Events.EventEmitter
+  , GitStats = require("git-stats")
+  , ParentSearch = require("parent-search")
+  , OneByOne = require("one-by-one")
+  , ChildProcess = require("child_process")
+  , Spawn = ChildProcess.spawn
+  , GitRepos = require("git-repos")
+  ;
+
+/**
+ * Importer
+ * Creates a new `Importer` instance.
+ *
+ * @name Importer
+ * @function
+ * @param {Object} options The `GitStatsImporter` options.
+ * @param {GitStats} gs The `GitStats` instance.
+ * @param {Object} data The commit data.
+ * @return {Importer} The `Importer` instance.
+ */
+function Importer(options, gs, data) {
+    this.path = options.path;
+    this.emails = options.emails;
+    this.ev = new EventEmitter();
+    this.gs = gs;
+    this.data = data;
+}
+
+/**
+ * stream
+ * Spawns the `git log` command and returns the stdout stream.
+ *
+ * @name stream
+ * @function
+ * @return {Stream} The stdout stream of `git log` command.
+ */
+Importer.prototype.stream = function () {
+    var args = ["log"]
+      , i = 0
+      ;
+
+    for (; i < this.emails.length; ++i) {
+        args.push("--author");
+        args.push(this.emails[i]);
```

Comment:
Oh, yeah––big thanks to @catcookie for [helping me to get the commits made by specific authors](http://stackoverflow.com/a/32567909/1420197). :sparkles: :cake: